pyproject.toml

```diff
@@ -112,27 +114,27 @@ default-tag = 'v999.0.0'
 # https://docs.astral.sh/ruff/rules/
 
 [tool.ruff]
-exclude = [
-  'examples',
-  'examples_old',
-  'tests'
-]
+exclude = ['examples', 'examples_old', 'tests_old']
 indent-width = 4
 line-length = 127
 
 [tool.ruff.format]
-docstring-code-format = true  # Whether to format code snippets in docstrings
-indent-style = 'space'  # PEP 8 recommends using spaces over tabs
-line-ending = 'lf'  # Line endings will be converted to \n
-quote-style = 'single'  # But double quotes in docstrings (PEP 8, PEP 257)
+docstring-code-format = true # Whether to format code snippets in docstrings
+indent-style = 'space'       # PEP 8 recommends using spaces over tabs
+line-ending = 'lf'           # Line endings will be converted to \n
+quote-style = 'single'       # But double quotes in docstrings (PEP 8, PEP 257)
 
 [tool.ruff.lint]
 select = [
-    'E9', 'F63', 'F7', 'F82',  # flake8 settings from existing CI setup
-    'E', 'W',  # pycodestyle
-    'F',  # Pyflakes
-    'I',  # isort
-    'S',  # flake8-bandit
+  'E9',
```

Comment:
Can you add comments on these undescribed flags?

Reply:
Done